pkgs/development/tools/scss-lint/default.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, bundlerApp }:
+
+bundlerApp {
+  pname = "scss_lint";
```

Comment:
You still have the underscore here? Is it bundix forcing it because apparently it's named this way on RubyGems?

Reply:
Yeah, it won't work without the underscore there. It used to be called `scss-lint`, but apparently the convention in ruby land is for an underscore.

Reply:
Mokay..